docs/changehistory/NextVersion.md

```diff
@@ -2,4 +2,3 @@
 publish: false
 ---
 # NextVersion
-
```

Comment:
just want to confirm there were no changes in nextver.md that got dropped when you created the merge

Reply:
There was a change in next version.md from one of Affan's PRs. However, that should be included in nextVersion.md in the release/4.4.x branch. So as long as its there, 4.4.0.md will be properly documented

Reply:
please make sure its there in the nextver.md in the 4.4. branch

Reply:
Yes, it has been incorporated in the other PR merging into release/4.4.x: https://github.com/iTwin/itwinjs-core/pull/6405/files#diff-4a7785af3b56bc69d2ad9f34257cd4a96cba53855aa039abeb761613d0f81988